homeassistant/components/media_player/sonos.py

```diff
@@ -595,6 +596,8 @@ def update(self):
 
                 if self._queue is None and self.entity_id is not None:
                     self._subscribe_to_player_events()
+            favs = self._player.get_sonos_favorites().get('favorites', [])
```

Comment:
Is this guaranteed to be populated? 

If I don't have a favorites list, will favourites exist in the dict?